test/parallel/test-inspector-open-dispose.mjs

```diff
@@ -8,37 +8,31 @@ common.skipIfInspectorDisabled();
 if (process.env.BE_CHILD) {
   await beChild();
 } else {
-  let firstPort;
-
   const filename = url.fileURLToPath(import.meta.url);
   const child = fork(filename, { env: { ...process.env, BE_CHILD: 1 } });
 
   child.once('message', common.mustCall((msg) => {
     assert.strictEqual(msg.cmd, 'started');
+    assert.strictEqual(msg.url, undefined);
 
-    child.send({ cmd: 'open', args: [] });
+    child.send({ cmd: 'open' });
     child.once('message', common.mustCall(wasOpenedHandler));
   }));
 
   function wasOpenedHandler(msg) {
     assert.strictEqual(msg.cmd, 'url');
-    const port = url.parse(msg.url).port;
+    const { port } = url.parse(msg.url);
     ping(port, common.mustSucceed(() => {
-      // Inspector is already open, and won't be reopened, so args don't matter.
       child.send({ cmd: 'dispose' });
       child.once('message', common.mustCall(wasDisposedWhenOpenHandler));
-      firstPort = port;
     }));
   }
 
-  function wasDisposedWhenOpenHandler(msg) {
+  async function wasDisposedWhenOpenHandler(msg) {
     assert.strictEqual(msg.cmd, 'url');
     assert.strictEqual(msg.url, undefined);
-    ping(firstPort, (err) => {
```

Comment:
We were pinging to see the port was closed, which meant if anything else caught the port, the test failed.

We don't really need this, since `inspector.url()` will be `undefined` after dispose, and `msg.url` is the result of `inspector.url()`

Reply:
Maybe we still want this assertion - but move this test to sequential to make sure nothing else listens to the port at this point?

Reply:
@MoLow Do we really want/need to assert this?
Isn't asserting `dispose` caused the `inspector.url()` to be `undefined` enoughe? We don't need to check the inspector actually does something, just that dispose clears it out...

Reply:
why? if `inspector.url()` is `undefined` and the port is still listening - that will be a bug

Reply:
@MoLow it would be a bug - but it would be a bug with the `inspector.url()`, not with the dispose...

Reply:
I disagree with that statement - I expect `dispose` to close the port

Reply:
@MoLow I am trying to avoid dependency on the port, do you think spying on calls to `inspector.close` would do?

Reply:
why not move this test to sequential?

Reply:
I'll move it 🙂 
Are you sure this would solve the issue?

Reply:
hopefully